pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -188,7 +188,7 @@ type ReconcileCnsFileAccessConfig struct {
 // Note:
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-func (r *ReconcileCnsFileAccessConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcileCnsFileAccessConfig) Reconcile(recCtx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	ctx, log := logger.GetNewContextWithLogger()
```

Comment:
Not directly related to this change but if the function is taking in a context should we use that instead of getting a new context here? This applies for all reconcilers